src/dynamics/models/elasticity_corotated_linear.rs

```diff
@@ -17,6 +17,21 @@ impl ConstitutiveModel for CorotatedLinearElasticity {
         )
     }
 
+    // https://www.math.ucla.edu/~cffjiang/research/mpmcourse/mpmcourse.pdf#subsection.6.3
```

Comment:
```suggestion
    // https://www.math.ucla.edu/~cffjiang/research/mpmcourse/mpmcourse.pdf#subsection.6.3
    // This is essentially the same calculation as `[pos_energy]` but without ignoring the negative part.
```

Reply:
I added a reference to the `pos_energy` instead.